tools/alpine-net/Makefile

```diff
@@ -0,0 +1,27 @@
+APP_NAME = alpine-net
+IMG = $(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/$(APP_NAME)
+TAG = $(DOCKER_TAG)
+binary=$(APP_NAME)
```

Comment:
`binary` is not needed as there is no building done for this tool.

Reply:
done